reproductions/NugetProfilerVersionMismatch.TypeLoadException/Program.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Npgsql;
+
+namespace HttpMessageHandler.StackOverflowException
```

Comment:
Forget to update the namespace?

Reply:
Good catch, fixed in c844ff61aafcbdc661212afdf6fa582a5c8b7cc8